packages/pluginval/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=pluginval
+_plugin_uri='https://github.com/Tracktion/pluginval/'
+pkgver=1.0.3
+pkgrel=1
+pkgdesc='Cross platform DAW plugin testing and validation tool'
+arch=(x86_64 aarch64)
+url='https://github.com/xunil-cloud/CloudReverb'
+license=(GPL-3.0-only)
+depends=(freetype2 glibc gcc-libs)
+makedepends=(cmake webkit2gtk)
+groups=(pro-audio)
```

Comment:
Not sure whether it makes sense to put this package into the `pro-audio` group?

Reply:
Sure, it's certainly related to pro-audio plugins :)